packages/mds-logger/README.md

```diff
@@ -1,3 +1,28 @@
 # MDS Logger
 
 Logging interface to be used for all MDS packages. Provides structured JSON logging so that logs can be parsed and indexed by an external log aggregator, and redacts sensitive information (e.g. lat/lng) from logs.
+
+## Debug Logging
+
+### Writing Debug Logs
+
+The mds-logger debug method behaves quite similarly to the [debug](https://www.npmjs.com/package/debug) module, with the only difference being the default namespace of `mds`, and prefixing of any custom namespaces with `mds:${custom_namespace}`.
+
+Examples:
+
+```typescript
+import logger from "@mds-core/mds-logger";
+
+// writing with the default namespace
+logger.debug()("Some super debug message");
+
+// writing with a custom namespace
+logger.debug("agency")("Some super duper debug message from agency!");
+
+// writing out with log data
+logger.debug("agency")("some log message with data", { foo: "bar" });
```

Comment:
I just noticed this, but typically the way node's `debug` is used is that it's called once to get a logger instance and then used. So, it would typically be used within a module like:
```ts
const agencyDebug = logger.debug('agency')
agencyDebug("Some super duper debug message from agency!")
agencyDebug("some log message with data", { foo: "bar" })
```

I wonder if it might make some sense to create separate loggers. Something like:
`agency-logger.ts`
```ts
import logger from '@mds-core/mds-logger'
export default {
  ...logger,
  debug: logger.debug('agency')
}
```

Then mds-agency modules could:
```ts
import logger from './agency-logger'
```

Something like that anyway. You get the idea.

Reply:
I believe that the currently implemented API for `mds-logger` should support this use-case, yup ✅

Reply:
I agree. I just think that that should be documented as the preferred pattern. In fact, `mds-logger` could export a factory function that accepts a namespace and returns a logger object, such that an `agency-logger.ts` could simply:
```ts
import { CreateLogger } from '@mds-core/mds-logger'
export default CreateLogger('agency')
```
Good work.

Reply:
Definitely. I'll probably add a factory & update the examples when I cut a PR to switch everything over to the namespaced logging methods ✅